### DISCUSSION TITLE
Better to run SCENIC+ separately on individual samples, or on the integrated dataset?

### DISCUSSION THREAD

**Discussion by solvi808** [aertslab/scenicplus#403] [General]
I have 3 samples of the same condition (although there could be batch effects).

I have created an integrated dataset using Signac, but also analysed each sample individually where I have per-sample clusters.

So I am wondering if its better that I run SCENIC+ once on the integrated dataset or 3 times on the individual datasets and then try to find which regulons are conserved. Is bulk / batch going to cause some issues or reduce sensitivity, what is the ideal approach ?

**Comment by SeppeDeWinter** [comment]
Hi @solvi808 

It is very difficult to answer what is *best* in general without working on the data.

In general I would suggest to run SCENIC+ once on the integrated dataset. More data will give better results and you won't have to try to integrate results from different runs, which can be tricky.

Batch effects can be an issue, in our hand however this has never been a very big issue.

I hope this helps?

Best,

Seppe
